pandas/core/frame.py

```diff
@@ -5005,39 +5005,41 @@ def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
 
     def applymap(self, func):
         """
-        Apply a function to a DataFrame that is intended to operate
-        elementwise, i.e. like doing map(func, series) for each series in the
-        DataFrame
+        Apply a function to a Dataframe elementwise.
+
+        This method applies a function to a DataFrame that is intended to
+        operate elementwise, i.e. like doing map(func, series) for each series
```

Comment:
We should probably mark it as code  \``map(func, series)`\`

Reply:
thanks! addressed